### PR TITLE
Add NPM workspaces

### PR DESCRIPTION
**Pull request by leite08**
Ref https://github.com/metriport/metriport-internal/issues/143

### Dependencies

- Upstream: https://github.com/metriport/metriport/pull/60
- Downstream: TBD

### Description

Add NPM workspaces.

With this PR, we can do things like:
- `$metriport> npm install` to run `npm install` on all packages automatically
  - likewise for other commands, like `lint` as added on `package.json`
  - including commands that are optional on the packages (with `--if-present`)
- to run a command on a package from the root folder, one can issue `npm run <cmd> -w <package-name>`

### Release Plan

- anytime

### CODE REVIEW

**Comment by leite08** [comment]
See https://github.com/metriport/metriport-internal/pull/270

File: package.json [hunk 1]
```diff
@@ -3,14 +3,24 @@
...
     "prep-garmin": "cd api/lambdas/garmin/ && npm install && npm run test && cd ../../../",
     "prep-token-auth": "cd api/lambdas/token-auth/ && npm install && npm run test && cd ../../../",
-    "prep-connect-app": "cd connect-widget/app/ && npm install && npm run build && cd ../../../",
-    "prep-connect-app-staging": "cd connect-widget/app/ && npm install && npm run build:staging && cd ../../../",
-    "prep-deploy": "npm run prep-infra && npm run prep-api-app && npm run prep-token-auth && npm run prep-garmin && npm run prep-connect-app",
-    "prep-deploy-staging": "npm run prep-infra && npm run prep-api-app && npm run prep-token-auth && npm run prep-garmin && npm run prep-connect-app-staging"
+    "prep-deploy": "npm install && npm run build && npm run test && npm run prep-token-auth && npm run prep-garmin",
+    "prep-deploy-staging": "npm install && npm run build:staging && npm run test && npm run prep-token-auth && npm run prep-garmin",
+    "build": "npm run build --workspaces",
+    "build:staging": "npm run build -w api && npm run build:staging -w dash && npm run build -w infra",
```
Comment (Goncharo) [inline package.json:11]: dash? this is OSS

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (leite08) [reply package.json:11]: good catch, missed updating this one

**Review by Goncharo — APPROVED** [review]
🚢

**Pull request merged**